babel.config.js

```diff
@@ -18,6 +18,9 @@ module.exports = function(api) {
     '@babel/plugin-syntax-dynamic-import',
   ];
   const env = {
+    development: {
+      compact: false,
+    },
     test: {
       plugins: ['dynamic-import-node'],
```

Comment:
should it be here too?

what if we turned it off but added it in `production`? is there any place other than production to turn it on?

Reply:
My feeling is that we should run the tests on compact.